schemast/printer_test.go

```diff
@@ -104,6 +104,19 @@ func TestPrintAddImport(t *testing.T) {
 	require.Equal(t, matches, 1)
 }
 
+func TestPrintStructOnSameLine(t *testing.T) {
```

Comment:
Using `parser.ParseExp` breaks this check. Generated struct is put in file as
```
struct{
}{}
```

Do not know if this is non issue and we can live with or we would like to have formatting like before. All fields on one line.

Reply:
![image](https://user-images.githubusercontent.com/7361100/153880126-15cccd1d-4adf-4dd2-a652-22745510dc1e.png)